miqa/core/tasks.py

```diff
@@ -376,8 +377,17 @@ def perform_export(project_id: Optional[str]):
                     f'{frame_object.scan.name} not exported; this scan was uploaded, not imported.'
                 )
     export_df = pandas.DataFrame(data, columns=IMPORT_CSV_COLUMNS)
+
     try:
-        export_df.to_csv(export_path, index=False)
+        if export_path.endswith('csv'):
+            export_df.to_csv(export_path, index=False)
+        elif export_path.endswith('json'):
+            json_contents = import_dataframe_to_dict(export_df, project)
+            json.dump(json_contents, open(export_path, 'w'))
```

Comment:
```suggestion
            with open(export_path, 'w') as fd:
                json.dump(json_contents, fd)
```

That will explicitly ensure that we don't leak file descriptors.